src/UrqlClient.re

```diff
@@ -511,6 +511,32 @@ let query =
   |> Wonka.toPromise;
 };
 
+[@bs.send]
+external readQueryJs:
```

Comment:
So in this case w/ `readQueryJs` you seem to be writing a binding for `readQuery` but never actually using it; instead, `readQuery` below just re-implements the same method in Reason. I actually think the latter is nicer – it takes advantage of the niceties of Reason to do what `pipe` does in `urql`.

TL;DR I think you can get rid of L514 – 524 altogether 😄

Reply:
Yes that makes sense. Actually I've been influenced by the `queryJs` and `mutationJs` ones. If I'm not mistaken, those are not used either. What's the reason to keep them then? Is it because one version returns a `Wonka.sourceT` while the other returns a `promise`?

Reply:
Ah that's actually a good shout, they can be removed as well! Good eye 👀